src/WorkOS.net/Services/DirectorySync/Entities/User.cs

```diff
@@ -20,6 +20,12 @@ public class User
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        /// <summary>
+        /// The User's identity provider's identifier.
```

Comment:
Can we use "directory provider" here to be consistent with the [docs](https://workos.com/docs/reference#directory-user-idp_id)?

```suggestion
        /// The User's directory provider's identifier.
```

Reply:
Done!